src/hooks/use-pull-push-timestamps.ts

```diff
@@ -1,92 +1,97 @@
 import { __, sprintf } from '@wordpress/i18n';
 import { formatDistanceToNow } from 'date-fns';
-import { useCallback, useState, useEffect } from 'react';
+import { useCallback, useState } from 'react';
+import { getIpcApi } from '../lib/get-ipc-api';
+import { SyncSite } from './use-fetch-wpcom-sites';
 
-const SYNC_TIMESTAMPS_STORAGE_KEY = 'wp-studio-sync-timestamps';
+export function usePullPushTimestamps() {
+	const [ tooltips, setTooltips ] = useState< Record< string, string > >( {} );
```

Comment:
The tooltip text does not accept promises so I am storing it like this

Reply:
in an ideal world, we could access `connectedSite.lastPullTimestamp` and `connectedSite. lastPushTimestamp` without the need of fetching the information from the file.

the callbacks shouldn't need to access the siteId and connectedSiteId, and it would receive the whole connectedSite or directly the timestamp.

I didn't test it, but ideally we use that information directly and every time we need to update the timestamp, we save it to the fiel and also update the apps state.

Reply:
Did a bunch of refactor in https://github.com/Automattic/studio/pull/714/commits/ebf3a4bec7a112cdba42c89a9a80b18922c009e3 to simplify data flow 👍